HISTORY.md

```diff
@@ -9,9 +9,12 @@ All notable changes to this project will be documented in this file.
     This is a follow-up of [#193].
 * Tests
   * Moved integration tests to a dedicated space and updated documentation for it. (via [#260]) 
+* Changed
+  * If 'homepage' property is solely a period, then omit 'website' entry from the externalReferences array. (via [#264])
```

Comment:
thanks for adding the note. 
please move it to the top of the list, as a change is more important than some documentations or tests - at least in the scope of a changelog.

Reply:
I have moved it to the top of the changelog for the next release, above docs and tests.